### ISSUE TITLE
Redux FAQ: Organizing State Outdated

### ISSUE THREAD

**Issue by wadsworj** [reduxjs/redux#4421]
## What docs page needs to be fixed?

- **Faq**:
- **Redux FAQ: Organizing State**:

https://redux.js.org/faq/organizing-state

## What is the problem?

I was looking at this today and this FAQ seems outdated.

In particular, it links to two community packages that have been out of date since 2015:

[redux-component](https://github.com/tomchentw/redux-component), [redux-react-local](https://github.com/threepointone/redux-react-local)

It also makes mention of this.setState().

There is no mention of RTK, useSelector, or react hooks.

As a beginner, this is a very confusing page to come across via Google.

## What should be changed to fix the problem?

Ideally - it should be updated. If not - it should be marked as depreciated or removed.

**Comment by markerikson** [comment]
Yeah, I wrote the original FAQ page back in 2016, and most of it hasn't been updated since then. Unfortunately, I'm busy enough that I haven't had time to go through and clean it up.  Sorry!

The general principles in the FAQ are still 100% valid, but agreed that some of the _specific_ bits of advice and links are out of date.

If anyone else would like to help sweep through the FAQ entries and update things, I'd really appreciate it!

**Comment by dhananjaysa92** [comment]
Looking into this issue
